src/main/kotlin/net/atos/zac/identity/IdentityService.kt

```diff
@@ -60,4 +67,22 @@ class IdentityService @Inject constructor(
             .map { it.toUser() }
             .sortedBy { it.getFullName() }
     }
+
+    fun checkIfUserIsInGroup(userId: String?, groupId: String?) {
+        if (userId == null || groupId == null) {
+            LOG.fine { "Missing user id '$userId' or group id '$groupId'. Cannot check is user is in group!" }
+            return
+        }
+
+        val isUserInGroup = listUsersInGroup(groupId)
+            .map { it.id }
+            .contains(userId)
+
+        if (!isUserInGroup) {
+            LOG.warning(
```

Comment:
We should not log this as a warning. It's a 4xx validation error and we should never log these as warnings, or at all really. Warnings and errors should be resolved for real system errors. Because 4xx errors are caused by end users / frontend clients over which we have no control. In this case we should not log anything at all.